_templates/troubleshooting.md

```diff
@@ -0,0 +1,113 @@
+---
+title: [Title]
+description: [Brief description of the How-To article]
+type: [Troubleshooting]
+teampulseid: [number]
+ticketid: [number]
+pitsid: [number] <!-- Not sure we need this, as pits item are related to teampulse anyway -->
+<!-- TO DO: What else should we have here? -->
+---
+
+# Title
```

Comment:
Same concern as above applies here. Do we want to generate this automatically or do we want to provide it manually?